src/disco/purify.js

```diff
@@ -0,0 +1,13 @@
+import purify from 'core/purify';
```

Comment:
I wonder if there's a use-case for the external link treatment in amo in the future? That might mean we'd want two purify instances centrally rather than splitting the usage by application.

Reply:
I find very intrusive to force `target="_blank"` on links in general, and I don't think we have [valid use cases](https://css-tricks.com/use-target_blank/) for them on AMO. That's why I ended up with this separation.

Reply:
@willdurand yeah I agree with that - I can't see a use-case at the moment.